scripts/gen-ocaml

```diff
@@ -1,12 +1,59 @@
 #! /usr/bin/env bash
 #
-# Generate OCaml parsers.
+# Generate OCaml parsers
+#
+# For lang=ruby, the file hierarchy we create looks like this:
+#
+# ocaml-src
+# ├── bin
+# │   ├── dune
+# │   └── Main.ml
+# ├── lib
+# │   ├── CST.ml
+# │   ├── dune
+# │   ├── Parse.ml
+# │   ├── parser.c
+# │   ├── scanner.cc
+# │   └── tree_sitter
+# │       └── parser.h
+# └── tree-sitter.opam
```

Comment:
why not tree-sitter-ruby.opam? We will not get conflicts?

Reply:
Well-spotted. This was indeed a problem. It's already fixed but I didn't update this comment.